src/components/structures/MatrixChat.tsx

```diff
@@ -350,10 +350,8 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
 
         if (SettingsStore.getValue("analyticsOptIn")) {
             Analytics.enable();
-            CountlyAnalytics.instance.enable(false);
-        } else {
-            CountlyAnalytics.instance.enable(true);
         }
+        CountlyAnalytics.instance.enable(true); // anonymous
```

Comment:
Let's use the style we have elsewhere for annotating random booleans:

```suggestion
        CountlyAnalytics.instance.enable(/* anonymous = */ true);
```

Reply:
![image](https://user-images.githubusercontent.com/2403652/97716778-f1086d80-1abb-11eb-81b3-9e6d3f50d8c4.png)


Guess I'm spoiled :P

Reply:
Ah yeah, my editor doesn't seem to do that at the moment 😜